ParisBNF/abb/BNFabb28.xml

```diff
@@ -0,0 +1,312 @@
+                         <desc type="StampExlibris">Stamp of the Library</desc>
+                     </item>
+
+                     <item xml:id="e4">
+                        <locus target="#1v"/>
+                        <desc type="Unclear">A few words written in a small script, partly erased</desc>
+                    </item>
+                       
+                       <item xml:id="e5">
+                          <locus target="#2r"/>
+                          <desc type="StampExlibris">Shelfmark and stamp of the Library</desc>
+                       </item>
+                       
+                       <item xml:id="e6">
+                          <locus from="2r" to="86v"/>
+                          <desc type="Correction">Omissions inserted interlineally or in the margins, on some folia, also by means 
+                             of <foreign xml:lang="am">tamallas</foreign>-signs</desc>
+                         </item>
+                       
+                         <item xml:id="e7">
+                          <locus target="#70v"/>
+                          <desc type="Unclear">One isolated word written, possibly pen trial or writing exercise</desc>
+                       </item>
+
+                    <item xml:id="e8">
+                      <locus target="#87v #88r"/>
+                       <desc type="StampExlibris">Stamps of the Library</desc>
+                   </item>           
+
+                      </list>
+                </additions>
+                  <bindingDesc>
+                     <binding contemporary="true" cert="medium">
+                        <decoNote xml:id="b1">Two wooden boards.</decoNote>
+                        <decoNote xml:id="b2" type="Cover">Blind-tooled leather cover.</decoNote>
+                        <decoNote xml:id="b3" type="Boards">Textile inlays.</decoNote>
+                        <decoNote xml:id="b4" type="SewingStations">4</decoNote>
+                           <decoNote xml:id="b5" type="Endbands" cert="low">Endbands.</decoNote>
+                           <decoNote xml:id="b6" type="Other">                 
+                                    <material key="wood"/>
+                                    <material key="leather"/>
+                                    <material key="textile"/>
+                              </decoNote>
+                    </binding>
+                  </bindingDesc>
+               </physDesc>
+               <history>
+                  <origin>
+                     <origDate notBefore="1700" notAfter="1799">18th century, an earlier period (17th century) cannot be excluded.</origDate>
+                     <ref target="#a3">Additional note 3</ref> suggests that the manuscript is related to the <persName ref="ETH1274Betaes">Beta ʾƎsrāʾel</persName> community.
+                     <bibl><ptr target="bm:Butts2013Reception"/><citedRange unit="page">5-6</citedRange></bibl> indicates that, contrary to R. Cowley's claim and what may
```

Comment:
reference to bm:Cowley1984Commentaries? https://www.jstor.org/stable/pdf/41299680.pdf here page 6 footnote 10 
![image](https://user-images.githubusercontent.com/17987728/232738463-312d4490-c0fa-48d9-b9fa-47b432a8284b.png)

Reply:
see also 
![image](https://user-images.githubusercontent.com/17987728/232738210-674f51b6-19db-4d47-91b7-ccb4ffd503d7.png)

Reply:
I limited the indications to Butts 2013 as it refers also to Cowley quite clearly and may be considered the enquiry's latest stage. The second reference fits more the work entry (I will have a look at it again).